components/script/dom/document.rs

```diff
@@ -247,6 +248,9 @@ pub struct Document {
     referrer: Option<String>,
     /// https://html.spec.whatwg.org/multipage/#target-element
     target_element: MutNullableHeap<JS<Element>>,
+    /// https://w3c.github.io/uievents/#event-type-dblclick
+    #[ignore_heap_size_of = ""]
```

Comment:
I could remove the #event-type-dblclick comment if you think it isn't needed? I was just mirroring the style of the other field declarations when adding this.
Although removing the 
 ```rust
#[ignore_heap_size_of = ""]
```
gives the error
``the method 'heap_size_of_children' exists but the following trait bounds were not satisfied: `core::option::Option<(std::time::Instant, euclid::TypedPoint2D<f32, euclid::UnknownUnit>)> : heapsize::HeapSizeOf```